frontend/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb",
+        "prettier",
```

Comment:
should be last in this list

Reply:
yes sir, prettier moved in the end of this list